.cloudbuild/scripts/cmd/integration-tests/main.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	gomodcacheDir = ".gomodcache-ci"
+	gocache       = "/builder/home/.cache/go-build"
```

Comment:
My 2c: let's place _all_ caches under the same temp dir, flip the corresponding variables (GOCACHE, GOMODCACHE) and chown that entire dir once.

WDYT?